internal/client/network/types/multiaddr/multiaddr.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2025 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package multiaddr
+
+import (
+	libp2p "github.com/libp2p/go-libp2p/core"
+)
+
+// Multiaddr type used in Gossamer
+type Multiaddr struct {
```

Comment:
why Multiaddr wrapped in Multiaddr?

Reply:
We will eventually need to implement some public methods on this type in future work (see [substrate impl](https://github.com/paritytech/polkadot-sdk/blob/414a8fc2eda3bb72e30cefdba628cf6c361cd6e1/substrate/client/network/types/src/multiaddr.rs#L44)).  Just laying the groundwork for now.